### DISCUSSION TITLE
DreamBooth on Automatic 1111: Exception training model: 'No executable batch size found, reached zero.'.

### DISCUSSION THREAD

**Discussion by drfraktal** [d8ahazard/sd_dreambooth_extension#787] [General]
Hi guys, I set up DreamBooth on Automatic 1111 and always got this error: 

Exception training model: 'No executable batch size found, reached zero.'.

Any idea what's the problem? Thanks in advance!

**Comment by teebu** [comment]
Same here. Latest version. 12gb card. Looks like its running out of memory.

**Comment by TypicalPlay** [comment]
Same error here.
12gb card. Was working well earlier.

RuntimeError: CUDA out of memory. Tried to allocate 114.00 MiB (GPU 0; 12.00 GiB total capacity; 11.13 GiB already allocated; 0 bytes free; 11.28 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
Steps:   0%|

**Comment by 78Alpha** [comment]
That would be accelerate trying to find a batch size your card can run but not finding one due to memory limits.

You may have to revert back or update to a known low memory commit.

**Comment by coudys** [comment]
I have 8GB VRAM GTX 1070 and I managed to find two settings (difference only EMPTY vs [filewords] in Instance Prompt and Class prompts) that work for me. Depending on how many dataset images I have I can use [filewords] for Instance and Class prompts. If I get this error I have to leave them both blank:
created custom Model using v1-5-pruned-emaonly.safetensors

> Dreambooth tab:

	Input Tab: 
		Settings Tab:
				Use Lora: checked
				Batch Size: 1 (default)
				Set Gradients to None When Zeroing: checked (default)
				Gradient Checkpointing: checked (default)
				Lora UNET Learning Rate: 0.0001 (default)
				Lora Text Encoder Learning Rate: 0.00005 (default)
				Learning Rate Scheduler: constant
				Optimizer: 8Bit AdamW (default)
				Mixed Precision: Fp16
				Cache Latents: unchecked
				Train UNET: unchecked
				Pad Tokens: unchecked
				Shuffle Tags: unchecked
				
		Concepts Tab:
			Concept1:
				Classification Dataset Directory: used repository from https://github.com/Luehrsen/sd_regularization_images
				Instance Token: Emmawatsonexample
				Class Token: a woman
				Instance Prompt: empty (can try [filewords] or "a photo of Emma", sometimes it works)
				Class Prompt: empty (can try [filewords] or "a photo of a woman", sometimes it works)
				Class Images Per Instance Image: 0
			
all other settings are default
Can someone tell me a workflow or point to a clear guide on how to extract Lora from a created checkpoint file? 2GB > 147MB?
